### ISSUE TITLE
cannot run in ubuntu 18.0

### ISSUE THREAD

**Issue by a524631266** [Almenon/AREPL-vscode#128]
today , I download your "arepl for python" extension in ubuntu 18.0,but when i keydown "ctrl" +"shift" +"a",
the vs call the error "command 'extension.currentAREPLSession' not found"!!,But in window system ,it runs well!   [err picture](https://github.com/a524631266/AREPL-vscode/blob/master/ubuntu18.0.err.png)
![test1111](https://user-images.githubusercontent.com/14838935/46914573-528f3980-cfd2-11e8-9718-de800454ec9a.png)

**Comment by a524631266** [comment]
I test my two ubuntu computer,and find the same err in these two computer

**Comment by a524631266** [comment]
I have solved this error.Ubuntu system cannot install this package through the vs market ,i git pull the source,and vsce package.the install the arepl-1.0.2.vsix

https://github.com/a524631266/AREPL-vscode/blob/master/arepl-1.0.2.vsix

**Comment by Almenon** [comment]
Thanks for reporting this.  It's a case issue.... again.  This is very frustrating - I thought I solved this for good in #43 when I turned on forceConsistentCasingInFileNames.

But apparently forceConsistentCasingInFileNames will only stop you if you have a different case in your source code - if there is a different case in your out folder it doesn't care. So when I used my old laptop to release v2 it had Utilities.js in the out folder instead of utilities.js, causing the error you ran into.

Because I can't rely on forceConsistentCasingInFileNames I'll write a unit test for launching the extension in a linux box - that should catch any future errors :) (https://github.com/Almenon/AREPL-vscode/issues/129 )
